hs/src/Reach/Eval/Core.hs

```diff
@@ -2663,7 +2664,7 @@ mustBeObjectTy err = \case
 mustBeUIntTy :: DLType -> App DLType
 mustBeUIntTy ty = case ty of
   T_UInt {} -> return ty
-  _ -> expect_ $ Err_Type_Mismatch (T_UInt UI_Word) ty
+  _ -> expect_ $ Err_Type_Mismatch (T_UInt UI_Word) ty Nothing Nothing
```

Comment:
Push the args back onto `mustBeUIntTy`?

Reply:
Ah, I dropped some branching paths in my quest to push args up to where they needed to be used.